### PR TITLE
Colin dev

### PR DESCRIPTION
**Pull request by colindaven**

### CODE REVIEW

**Comment by LisaHollstein** [comment]
I think the flowchart could be more clear. All those unlabelled arrows are a bit confusing

File: growth_rate/run_bed_to_csv.sh [hunk 1]
```diff
@@ -57,9 +59,9 @@ count_bam_txt=`ls -1 ${bam%bam}bam.txt 2>/dev/null | wc -l`
 #fi
 
 # check if input exists
-count_bam=`ls -1 ../*calmd.bam 2>/dev/null | wc -l`
-count_bai=`ls -1 ../*calmd.bam.bai 2>/dev/null | wc -l`
-count_bam_txt=`ls -1 ../*calmd.bam.txt 2>/dev/null | wc -l`
+count_bam=`ls -1 /*calmd.bam 2>/dev/null | wc -l`
```
Comment (LisaHollstein) [inline growth_rate/run_bed_to_csv.sh:62]: you left the `/`

File: nf_wochenende.nf [hunk 2]
```diff
@@ -133,7 +133,7 @@ workflow {
     // multiqc
     //multiqc(bam_stats.out.collect(), bam_stats.out)
 
-    // metagen window filter
+    // metagen window filter - @Lisa - I think this step is covered in plots and or growth rate so might not be needed?
```
Comment (LisaHollstein) [inline nf_wochenende.nf:136]: As far as I can see, this exact stept, which filters out human or mouse reads and creates the `.filt.sort.csv`, doesn't seem to be covered by plots yet

Comment (LisaHollstein) [reply nf_wochenende.nf:136]: Oh, you added that part... then we can of course get rid of this extra process

Comment (colindaven) [reply nf_wochenende.nf:136]: ok, pls remove

File: growth_rate/run_bed_to_csv.sh (hunk 1 above)
Comment (colindaven) [reply growth_rate/run_bed_to_csv.sh:62]: true, that's bad

**Comment by colindaven** [comment]
> I think the flowchart could be more clear. All those unlabelled arrows are a bit confusing

Yes, very true. We can check it at the end, or make it prettier later. Maybe this demonstrates we have a lot of unused code and processes at present, or just many outputs from the wochenende process which aren't further used.

**Review by LisaHollstein — APPROVED** [review]

**Pull request merged**